src/components/ciPipeline/SourceMaterials.tsx

```diff
@@ -120,11 +123,17 @@ export const SourceMaterials: React.FC<SourceMaterialsProps> = function (props)
                 if (props.ciPipelineSourceTypeOptions.length == 1) {
                     selectedMaterial = props.ciPipelineSourceTypeOptions[0]
                 } else {
-                    selectedMaterial = props.ciPipelineSourceTypeOptions.find((i) => i.isSelected === true)
+                    selectedMaterial =
+                        props.ciPipelineSourceTypeOptions.find((i) =>
+                            i.value === SourceTypeMap.WEBHOOK ? i.isSelected : i.value === mat.type,
+                        ) || props.ciPipelineSourceTypeOptions[0]
                 }
-
                 let errorObj = props.validationRules?.sourceValue(mat.value)
-
+                {
+                    console.log(mat)
+                }
+                const isBranchRegex = mat.type === SourceTypeMap.BranchRegex
+                const isBranchFixed = mat.type !== SourceTypeMap.WEBHOOK && mat.type === SourceTypeMap.BranchFixed
```

Comment:
we can remove mat.type !== SourceTypeMap.WEBHOOK check

Reply:
Done